docs/filters.rst

```diff
@@ -0,0 +1,87 @@
+.. _filters:
+
+Filtering
+=========
+
+REST resource filtering is provided via method ``_filter_queryset``. If you want implement your own filter technique
+you can add here your own code. Input params of the method are only ``qs`` which represents data that you want to filter
+(for model resources it is django QuerySet object).
```

Comment:
Dovoluji si navrhnout učesanou verzi:
REST resource filtering is provided via the ``_filter_queryset`` method. 
If you want to implement your own filtering technique, you can override it.
Its single argument ``qs`` contains data that you will filter (an instance of Django QuerySet in case of a model resource).